lib/improver/nbhood/nbhood.py

```diff
@@ -221,7 +221,7 @@ def process(self, cube, mask_cube=None):
         if np.isnan(cube.data).any():
             raise ValueError("Error: NaN detected in input cube data")
 
-        cubelist = iris.cube.CubeList([])
+        cubelist = []
```

Comment:
is it confusing to call a list of cubes a cubelist?

Reply:
Updated